libs/blocks/notification/notification.css

```diff
@@ -442,12 +453,17 @@
     gap: var(--spacing-m);
   }
 
-  .notification:is(.full-width, .ribbon) .foreground.container {
+  .notification:is(.full-width, .max-width-10-desktop) .foreground.container {
+    inline-size: unset;
+  }
+
+  .notification.full-width .foreground.container {
```

Comment:
On the topic of `.full-width`
Was it discussed to rename this variant? 
It just has other global implications that we should consider.
Masonry and ups both have this as a `1fr` backdoor to author content full col-span.

Reply:
Oh interesting -- no, "full-width" was the intention, but I was unaware we were trying to move away from that. Would `.max-width-12` make the most sense in that case, since we're already using similar elsewhere?

Reply:
Yes `max-width-12` would make more sense IMO especially since it's has the auto of `max-width-8` as default foreground container width. If you could decouple .full-width completely and let that only live in the grid functionality, that would be good. Related: I have a future task to support this in the text block but it would need a new variant like `full-width-grid` etc to support regression, but in your case since this is essentially net new, we should nip this in the bud if possible.